### ISSUE TITLE
U.S. Geological Survey Kelp Groves of the Pacific Coast and Islands Maps (kelp-maps)

### ISSUE THREAD

**Issue by wickr** [OregonDigital/OD2-migration#167]
**Item Count:** 49 items  
(5 others are in Oregon Maps primary set)

**Item Types:** Image

**Access Restrictions:** 0

**Complex Objects:** 0

**pid List:** https://github.com/OregonDigital/OD2-migration/blob/master/kelp-maps/kelp-maps_pids.txt

**Comment by carakey** [comment]
Fail - problem with Subject term "Kelps" (LCSH), expected on all works in collection. "Kelps" does not appear at all in the topic facet or on work show pages.

Investigation shows this on the edit page:
<img width="582" alt="image" src="https://user-images.githubusercontent.com/17655103/179617408-5f74ef36-90d3-4ca2-a290-12d155acdf5e.png">

In the screenshot, the URI paired with "Pacific Coast (Or.)" in the first block (https://id.loc.gov/authorities/subjects/sh85071892) is the URI for "Kelps", and the URI in the second block (http://id.loc.gov/authorities/subjects/sh87002619) is for "Pacific Coast (Or.)".

Downloaded CSV metadata shows the value for subject as "Pacific Coast (Mexico)|No label found [http://id.loc.gov/authorities/subjects/sh85071892]".

**Comment by wickr** [comment]
Not sure how this happened, but Kelps had been fetched from LOC, but we only had 2 statements in blazegraph:
```
<http://id.loc.gov/authorities/subjects/sh85071892> <http://www.loc.gov/mads/rdf/v1#authoritativeLabel> "Kelps" .
<http://id.loc.gov/authorities/subjects/sh85071892> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2004/02/skos/core#Concept> .
```
and our code doesn't use mads:authoritativeLabel. I wiped out the statements from our blazegraph to get a new, clean fetch, and that got all the statements with the correct labels. After a reindex of all works in this collection, I think we're good to go.

**Comment by carakey** [comment]
Review notes:

* Preliminary approved works include 1 from Oregon Maps primary set
* Format field does not seem to be displaying on work show page (not a migration issue, values are there per facets and viewer)
* Publisher facet is not present on the sidebar

**Comment by carakey** [comment]
@wickr Ready for bulk review

**Comment by wickr** [comment]
Bulk reviewed the rest. 
50 are now available with the 1 from Oregon Maps #35

Format field I think is just on the FileSet/File page?  https://prod.oregondigital.org/concern/parent/df73c051t/file_sets/f0df73c051t#content

No additional content or updates expected for this in OD1.
